admin/module/tests/README.md

```diff
@@ -1,62 +1,81 @@
-It is not intended to be a full description of the test features that you can 
+This is the quick-start to CodeIgniter testing. Its intent is to describe what
+it takes to set up your module and get it ready to run unit tests.
+It is not intended to be a full description of the test features that you can
 use to test your module. Those details can be found in the documentation.
 
 ## Resources
+
 * [CodeIgniter 4 User Guide on Testing](https://codeigniter4.github.io/userguide/testing/index.html)
-* [PHPUnit docs](https://phpunit.readthedocs.io/en/8.5/index.html)
+* [PHPUnit docs](https://phpunit.de/documentation.html)
+* [Any tutorials on Unit testing in CI4?](https://forum.codeigniter.com/showthread.php?tid=81830)
 
 ## Requirements
 
-It is recommended to use the latest version of PHPUnit. At the time of this 
-writing we are running version 8.5.13. Support for this has been built into the 
-**composer.json** file that ships with CodeIgniter and can easily be installed 
+It is recommended to use the latest version of PHPUnit. At the time of this
+writing we are running version 9.x. Support for this has been built into the
+**composer.json** file that ships with CodeIgniter and can easily be installed
 via [Composer](https://getcomposer.org/) if you don't already have it installed globally.
 
-	> composer install
+```console
+> composer install
+```
 
-If running under OS X or Linux, you can create a symbolic link to make running tests a touch nicer.
+If running under macOS or Linux, you can create a symbolic link to make running tests a touch nicer.
 
-	> ln -s ./vendor/bin/phpunit ./phpunit
+```console
+> ln -s ./vendor/bin/phpunit ./phpunit
+```
 
-You also need to install [XDebug](https://xdebug.org/index.php) in order
-for code coverage to be calculated successfully.
+You also need to install [XDebug](https://xdebug.org/docs/install) in order
+for code coverage to be calculated successfully. After installing `XDebug`, you must add `xdebug.mode=coverage` in the **php.ini** file to enable code coverage.
 
 ## Setting Up
 
-A number of the tests use a running database. 
-In order to set up the database edit the details for the `tests` group in 
-**phpunit.xml**. Make sure that you provide a database engine that is currently running
-on your machine. More details on a test database setup are in the 
-*Docs>>Testing>>Testing Your Database* section of the documentation.
+A number of the tests use a running database.
+In order to set up the database edit the details for the `tests` group in
+**phpunit.xml**.
+Make sure that you provide a database engine that is currently running on your machine.
+More details on a test database setup are in the
+[Testing Your Database](https://codeigniter4.github.io/userguide/testing/database.html) section of the documentation.
 
 ## Running the tests
 
 The entire test suite can be run by simply typing one command-line command from the main directory.
 
-	> ./phpunit
+```console
+> ./phpunit
```

Comment:
How does this work on Linux? I don't think the executable ever ends up in the project root.

Reply:
We create a symlink.
See https://github.com/codeigniter4/CodeIgniter4/pull/6274/files/8fe0acad8f8eaf8ffdbd67b176ea818422842397#diff-090405bd51245535c38ef723337f3525aad361cde9778a6f903b71cf9d5f802dR28